code_schemes/s01mag03.json

```diff
@@ -0,0 +1,70 @@
+{
+  "SchemeID": "Scheme-01c0f582",
+  "Name": "S01MAG03",
```

Comment:
Not sure about the naming here. It serves its purpose, and I don't have a better suggestion given the names already given to the radio shows and to the new magazine flows, but we might want to think about standardising this across the pipeline and TextIt going forward.

Reply:
Is the magazine for radio show episode 3, or magazine number 3?

Reply:
This is for Magazine 3, Magazine 1-2 were combined with drama.